src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -543,20 +510,22 @@ def cli_cosmosdb_mongodb_collection_update(client,
                                                         account_name,
                                                         database_name,
                                                         collection_name)
-
     mongodb_collection_resource = MongoDBCollectionResource(id=collection_name)
-    mongodb_collection_resource.shard_key = mongodb_collection.shard_key
-    mongodb_collection_resource.indexes = mongodb_collection.indexes
+    mongodb_collection_resource.shard_key = mongodb_collection.resource.shard_key
+    mongodb_collection_resource.indexes = mongodb_collection.resource.indexes
 
     if _populate_mongodb_collection_definition(mongodb_collection_resource, None, indexes):
         logger.debug('replacing MongoDB collection')
 
+    mongodb_collection_create_update_resource = MongoDBCollectionCreateUpdateParameters(
+        resource=mongodb_collection_resource,
+        options={})
```

Comment:
Why do you want to pass options with {} here?

Reply:
options is a required keyword argument for this object, but we do not allow option updates via this CLI command. Currently, they can update the options by performing a throughput update.